Statistics/Histograms/Histogram.cs

```diff
@@ -345,6 +345,21 @@ public void AddObservationsToHistogram(double[] data)
         }
         public int FindBinCount(double x, bool cumulative = true)
```

Comment:
i guess the only reason we have findbincount as being public is because it is in the histogram interface (which seems unnecessary) it seems like this method should be private